src/pybind/mgr/dashboard/frontend/src/app/shared/components/language-selector/language-selector.component.html

```diff
@@ -1,11 +1,12 @@
 <div ngbDropdown
      display="dynamic"
      placement="bottom-right">
-  <a ngbDropdownToggle
+  <button ngbDropdownToggle
      i18n-title
-     title="Select a Language">
+     title="Select a Language"
+     class="dropdown-toggle-button">
     {{ allLanguages[selectedLanguage] }}
-  </a>
+  </button>
```

Comment:
Rather than changing the HTML element and then having to restyle the look & feel of the new element to match the former one, what about leveraging the power of ARIA role descriptors, like [this one](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role)?